src/dataflow-types/src/storage.rs

```diff
@@ -0,0 +1,60 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use bytes::BufMut;
+use prost::Message;
+
+use mz_persist_types::Codec;
+use mz_repr::proto::{TryFromProtoError, TryIntoIfSome};
+
+include!(concat!(env!("OUT_DIR"), "/mz_dataflow_types.storage.rs"));
+
+impl From<&Result<Row, DataflowError>> for proto_literal::ProtoSourceData {
+    fn from(x: &Result<Row, DataflowError>) -> Self {
+        use proto_source_data::Kind;
+        ProtoSourceData {
+            kind: Some(match x {
+                Ok(row) => Kind::Ok(row.into()),
+                Err(err) => Kind::Err(err.into()),
+            }),
+        }
+    }
+}
+
+impl TryFrom<proto_literal::ProtoSourceData> for Result<Row, DataflowError> {
```

Comment:
is `proto_literal` imported through this file's protobuf's imports? or is this missing an import?